qa/pull-tester/rpc-tests.sh

```diff
@@ -93,12 +94,12 @@ testScriptsExt=(
   'keypool.py'
   'receivedby.py'
   'rpcbind_test.py'
-# 'script_test.py'            # requires create_block functionality that is not implemented for zcash blocks yet
+#  'script_test.py'
   'smartfees.py'
   'maxblocksinflight.py'
-# 'invalidblockrequest.py'    # requires create_block functionality that is not implemented for zcash blocks yet
-# 'forknotify.py'
-# 'p2p-acceptblock.py'        # requires create_block functionality that is not implemented for zcash blocks yet
+  'invalidblockrequest.py'
+  'invalidblockposthalving.py'
+  'p2p-acceptblock.py'
```

Comment:
@cronicc or maybe @alsala, could you remind me why these comments were removed?

Reply:
These tests have always been disabled since the original Zcash implementation due to missing support in the test framework. @alsala added support in https://github.com/HorizenOfficial/zen/pull/342/files?file-filters%5B%5D=.py and fixed + enabled the tests.

Not sure why script_test.py was not made compatible, @alsala would know.